pkg/gofr/container/container_test.go

```diff
@@ -4,11 +4,11 @@ import (
 	"context"
 	"testing"
 
+	"go.uber.org/mock/gomock"
+
 	"github.com/gorilla/websocket"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"go.uber.org/mock/gomock"
```

Comment:
this line should remain, there is no reason to have a distinct block for gomock before the github.com packages